client/protocols/backup.py

```diff
@@ -97,10 +96,14 @@ def send_all_messages(self, access_token, message_list):
         num_messages = len(message_list)
 
         send_all = self.args.submit or self.args.backup
-        timeout = self.LONG_TIMEOUT if send_all else self.SHORT_TIMEOUT
+        if send_all:
+            timeout = None
```

Comment:
This `timeout` eventually gets passed to `self.send_messages`, which calls `urllib.request.urlopen` with the `timeout`. Calling `urllib.request.urlopen` with a timeout of None is technically undefined behavior (according to the [docs](https://docs.python.org/3/library/urllib.request.html#urllib.request.urlopen)). However, it seems like this [won't break anything](http://www.gossamer-threads.com/lists/python/bugs/1071333).